processing/src/main/java/io/druid/query/timeseries/TimeseriesBinaryFn.java

```diff
@@ -36,14 +36,17 @@
 {
   private final Granularity gran;
   private final List<AggregatorFactory> aggregations;
+  private final int limit;
```

Comment:
This parameter looks useless, please remove it.

Reply:
done